platform-api/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -312,6 +314,57 @@ public void remove(@ApiParam(value = "User ID")
         userDao.remove(id);
     }
 
+
+    /**
+     * Allow to check if current user has a given role or not. status <b>200</b> and {@link UserInRoleDescriptor} is returned by indicating if role is granted or not
+     *
+     * @param role
+     *         role to search (like admin or manager)
+     * @param scope
+     *         the optional scope like system, workspace, account.(default scope is system)
+     * @param scopeId
+     *         an optional scopeID used by the scope like the workspace ID if scope is workspace.
+     * @return {UserInRoleDescriptor} which indicates if role is granted or not
+     * @throws ServerException
+     *         when unable to perform the check
+     */
+    @ApiOperation(value = "Check role for the authenticated user",
+            notes = "Check if user has a role in given scope (default is system) and with an optional scope id. Roles allowed: user, system/admin, system/manager.",
+            response = UserInRoleDescriptor.class,
+            position = 7)
+    @ApiResponses({@ApiResponse(code = 200, message = "OK"),
+                   @ApiResponse(code = 500, message = "Internal Server Error")})
+    @GET
+    @Path("/isUserInRole")
```

Comment:
sure, I was following issue description until now. As we're in user api we can also remove user

so inRole would be ok ? only may let think that we can get all the roles and not check them